Solutions/Okta Single Sign-On/Analytic Rules/PhishingDetection.yaml

```diff
@@ -0,0 +1,40 @@
+id: 78d2b06c-8dc0-40e1-91c8-66d916c186f3
+name: Okta Fast Pass phishing Detection
+description: |
+  'This query detects cases in which Okta FastPass effectively prevented access to a known phishing website'
+severity: High
```

Comment:
If this is blocked activity should it be high severity?

Reply:
Thanks Pete. Modified.